src/stream/resource-locator.provider.js

```diff
@@ -41,12 +44,13 @@ function clbResourceLocatorProvider() {
  * @desc
  * resourceLocator service
  * @memberof module:clb-stream
- * @param {object} $q AngularJS injection
- * @param {object} $log AngularJS injection
- * @param {object} clbError AngularJS injection
+ * @param {object} $q        DI
+ * @param {object} $log      DI
+ * @param {object} $injector DI
+ * @param {object} clbError  DI
  * @return {object} the service singleton
  */
-function clbResourceLocator($q, $log, clbError) {
+function clbResourceLocator($q, $log, $injector, clbError) {
```

Comment:
Support for dynamic injection as to be able to provide at runtime module announced at config time.